### PR TITLE
Added 3*3 ticker component column

### PR DESCRIPTION
**Pull request by Rahul139-stack**
Added ticker component, data can also be managed by json datasource
image

### CODE REVIEW

File: application/backend/src/main/java/io/typerefinery/websight/models/components/Columnticker.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    @Getter
+    @Inject
+    @Default(values = "12.5k")
+    public String value;
+
+
+    @Getter
+    @Inject
+    @Default(values = "pi pi-database")
+    public String icon;
```
Comment (Harikarthyk) [inline application/backend/src/main/java/io/typerefinery/websight/models/components/Columnticker.java:33]: Please remove unused variables from here

File: application/backend/src/main/java/io/typerefinery/websight/models/components/Smallticker.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import javax.inject.Inject;
+import lombok.Getter;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.apache.sling.models.annotations.Default;
+import org.apache.sling.models.annotations.Model;
+import org.apache.sling.models.annotations.injectorspecific.SlingObject;
+
+@Model(adaptables = Resource.class, defaultInjectionStrategy = OPTIONAL)
+public class Smallticker {
```
Comment (Harikarthyk) [inline application/backend/src/main/java/io/typerefinery/websight/models/components/Smallticker.java:14]: Please follow proper naming convention 

Smallticker to SmallTicker

File: application/backend/src/main/resources/apps/typerefinery/components/content/columnticker/dialog/.content.json [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        },
+        "dataTab": {
+            "sling:resourceType": "wcm/dialogs/components/tab",
+            "label": "Data",
+            "value": {
+                "sling:resourceType": "wcm/dialogs/components/textfield",
+                "name": "value",
+                "label": "Value"
+            },
+            "indicatorType": {
```
Comment (Harikarthyk) [inline application/backend/src/main/resources/apps/typerefinery/components/content/columnticker/dialog/.content.json:27]: Please remove unwanted codes from here.

I can see few things which are not used.

File: application/backend/src/main/resources/apps/typerefinery/components/content/columnticker/smallticker/.content.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "group": "Typerefinery",
+  "sling:resourceType": "ws:Component",
+  "description": "This is CTA component for Howlite",
```
Comment (Harikarthyk) [inline application/backend/src/main/resources/apps/typerefinery/components/content/columnticker/smallticker/.content.json:4]: Please change the description

File: application/backend/src/main/resources/apps/typerefinery/components/content/columnticker/smallticker/script.js [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    } catch (error) {
+        smallTickerUpdateView(jsonValue, id,bgColor);
+    }
+  };
+
+  fetchAndUpdateView();
+}
+
+function smallTickerUpdateView(jsonValue, id,bgColor) {
+  const smalltickerHtmlWithJsonValue = `
```
Comment (Harikarthyk) [inline application/backend/src/main/resources/apps/typerefinery/components/content/columnticker/smallticker/script.js:19]: Follow the camel case standars for all the variables.

File: application/backend/src/main/resources/apps/typerefinery/components/content/columnticker/smallticker/script.js [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+    id,
+    bgColor
+  );
+}
+
+$(document).ready(function (e) {
+  Array.from(document.querySelectorAll("#smallticker")).forEach(
+    (smalltickerComponent) => {
+      var componentDataPathticker =
+        smalltickerComponent.getAttribute("data-path");
```
Comment (Harikarthyk) [inline application/backend/src/main/resources/apps/typerefinery/components/content/columnticker/smallticker/script.js:57]: Follow the camel case standars for all the variables.

File: application/backend/src/main/java/io/typerefinery/websight/models/components/Columnticker.java (hunk 1 above)
Comment (Rahul139-stack) [reply application/backend/src/main/java/io/typerefinery/websight/models/components/Columnticker.java]: Done

File: application/backend/src/main/java/io/typerefinery/websight/models/components/Smallticker.java (hunk 2 above)
Comment (Rahul139-stack) [reply application/backend/src/main/java/io/typerefinery/websight/models/components/Smallticker.java]: Done

File: application/backend/src/main/resources/apps/typerefinery/components/content/columnticker/smallticker/.content.json (hunk 4 above)
Comment (Rahul139-stack) [reply application/backend/src/main/resources/apps/typerefinery/components/content/columnticker/smallticker/.content.json]: Done

File: application/backend/src/main/resources/apps/typerefinery/components/content/columnticker/smallticker/script.js (hunk 6 above)
Comment (Rahul139-stack) [reply application/backend/src/main/resources/apps/typerefinery/components/content/columnticker/smallticker/script.js]: Done

File: application/backend/src/main/resources/apps/typerefinery/components/content/columnticker/smallticker/script.js (hunk 5 above)
Comment (Rahul139-stack) [reply application/backend/src/main/resources/apps/typerefinery/components/content/columnticker/smallticker/script.js]: Done

File: application/backend/src/main/resources/apps/typerefinery/components/content/columnticker/dialog/.content.json (hunk 3 above)
Comment (Rahul139-stack) [reply application/backend/src/main/resources/apps/typerefinery/components/content/columnticker/dialog/.content.json]: Done

**Pull request merged**